packages/core/src/execution-engine/workflow-execute.ts

```diff
@@ -972,6 +973,14 @@ export class WorkflowExecute {
 		return workflowIssues;
 	}
 
+	private getCustomOperation(node: INode, type: INodeType) {
+		const { resource, operation } = node.parameters ?? {};
```

Comment:
This breaks type safety if we have stricter TS checks. Also it's better to check for conditions at the start.

```suggestion
    if (!node.parameters || !type.customOperations) {
      return undefined;
    }
    
		const { resource, operation } = node.parameters;
```